external-import/zerofox/src/main.py

```diff
@@ -43,6 +43,10 @@ def __init__(self):
             feeds=os.environ.get("ZEROFOX_COLLECTORS", None),
             logger=self.helper.connector_logger,
         )
+        self.author = stix2.Identity(
+            name="ZeroFox Connector",
```

Comment:
It's an organization, it must represent your company. So call it “ZeroFox” for short.
The fact that it was a connector that created the data will be modeled by another concept in OpenCTI. You don't need to worry about it.
+
You need to add a predictive id (you know, with the id generation function). If you don't, you'll create a random ID every time, and it will keep piling up in the platform.